tests/ArgumentConverterTest.php

```diff
@@ -1,36 +1,27 @@
 <?php
 
-namespace Spatie\Ray\Tests;
+use function PHPUnit\Framework\assertEquals;
+use function PHPUnit\Framework\assertFalse;
+use function PHPUnit\Framework\assertNull;
+use function PHPUnit\Framework\assertTrue;
 
-use PHPUnit\Framework\TestCase;
 use Spatie\Ray\ArgumentConverter;
 
-class ArgumentConverterTest extends TestCase
-{
-    /** @test */
-    public function it_returns_null_for_null_values()
-    {
-        $this->assertNull(ArgumentConverter::convertToPrimitive(null));
-    }
+it('returns null for null values', function () {
+    assertNull(ArgumentConverter::convertToPrimitive(null));
+});
 
-    /** @test */
-    public function it_returns_int_values_unchanged()
-    {
-        $this->assertEquals(1, ArgumentConverter::convertToPrimitive(1));
-        $this->assertEquals(PHP_INT_MAX, ArgumentConverter::convertToPrimitive(PHP_INT_MAX));
-    }
+it('returns int values unchanged', function () {
+    assertEquals(1, ArgumentConverter::convertToPrimitive(1));
```

Comment:
Could you rewrite all assertions to `expect()` ?

Reply:
Hi @freekmurze, I'm planning to open more PRs, should I do it for the next PRs as well? Thanks!

Reply:
Yup!